common/interfaces/desktop_app_interface/usb_event.h

```diff
@@ -0,0 +1,61 @@
+/**
+ * @file    usb_event.h
+ * @author  Cypherock X1 Team
+ * @brief   USB Event APIs.
+ *          Describes all the logic for interfacing with USB Events.
+ * @copyright Copyright (c) 2023 HODL TECH PTE LTD
+ * <br/> You may obtain a copy of license at <a href="https://mitcc.org/"
+ * target=_blank>https://mitcc.org/</a>
+ */
+#ifndef USB_EVENTS
+#define USB_EVENTS
+
+/*****************************************************************************
+ * INCLUDES
+ *****************************************************************************/
+#include <stdbool.h>
+#include <stdint-gcc.h>
+
+#include "communication.h"
+
+/*****************************************************************************
+ * MACROS AND DEFINES
+ *****************************************************************************/
+
+/*****************************************************************************
+ * TYPEDEFS
+ *****************************************************************************/
+typedef struct usb_event {
```

Comment:
Soft suggestion to start names of the members with `usb_`

Maybe:
```
typedef struct {
  uint8_t usb_event_flag;
  En_command_type_t usb_cmd_id;
  uint16_t usb_msg_size;
  uint8_t *p_usb_msg;
} usb_event_t;
```

Reply:
I will add `_t`. But

the member names without `usb_event_` suffix seem better. Example:

```
usb_event_t usb_event;

usb_event.usb_event_flag = true;
usb_event.flag = true;
```

Reply:
changed